elasticsearch5/src/main/java/org/vertexium/elasticsearch5/ElasticsearchSearchIndexConfiguration.java

```diff
@@ -42,6 +42,7 @@
     public static final String IN_PROCESS_NODE_DATA_PATH = "inProcessNode.dataPath";
```

Comment:
Are these other paths needed anymore?

Reply:
The other paths are needed for ES 1.7. In ES 5, the three paths are combined into `path.home`.

Reply:
@mwizeman just pointed out this is in the ES5 package so those configs are not needed. just pushed the deletion